lunes_cms/cms/admins/discipline_admin.py

```diff
@@ -70,7 +70,7 @@ def save_model(self, request, obj, form, change):
             obj.creator_is_admin = request.user.is_superuser
         obj.save()
 
-    def get_action_choices(self, request):
+    def get_action_choices(self, request, default_choices=""):
```

Comment:
Alternatively we can disable `arguments-differ` here locally, for `default_choices=""` is just a place holder.

Reply:
Well, if the parameter exists, it could directly be passed to the super method, right? Then it will be useful at least :D